[2025-03-05T23:29:19.426+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T23:29:19.431+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_stock_price.cleanup_data manual__2025-03-05T23:29:14.557026+00:00 [queued]>
[2025-03-05T23:29:19.434+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_stock_price.cleanup_data manual__2025-03-05T23:29:14.557026+00:00 [queued]>
[2025-03-05T23:29:19.434+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-03-05T23:29:19.437+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): cleanup_data> on 2025-03-05 23:29:14.557026+00:00
[2025-03-05T23:29:19.441+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_stock_price', 'cleanup_data', 'manual__2025-03-05T23:29:14.557026+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/daily_stock_price.py', '--cfg-path', '/tmp/tmp8bemfvbh']
[2025-03-05T23:29:19.442+0000] {standard_task_runner.py:105} INFO - Job 76: Subtask cleanup_data
[2025-03-05T23:29:19.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=829) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T23:29:19.443+0000] {standard_task_runner.py:72} INFO - Started process 832 to run task
[2025-03-05T23:29:19.457+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_stock_price.cleanup_data manual__2025-03-05T23:29:14.557026+00:00 [running]> on host 01071d25f042
[2025-03-05T23:29:19.481+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_stock_price' AIRFLOW_CTX_TASK_ID='cleanup_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T23:29:14.557026+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T23:29:14.557026+00:00'
[2025-03-05T23:29:19.482+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T23:29:19.487+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-03-05T23:29:19.487+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-05T23:29:19.487+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-05T23:29:19.924+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T23:29:20.471+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T23:29:20.472+0000] {logging_mixin.py:190} INFO - Cleaned up rows
[2025-03-05T23:29:20.473+0000] {connection.py:788} INFO - closed
[2025-03-05T23:29:20.517+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-03-05T23:29:20.579+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T23:29:20.592+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T23:29:20.593+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=daily_stock_price, task_id=cleanup_data, run_id=manual__2025-03-05T23:29:14.557026+00:00, execution_date=20250305T232914, start_date=20250305T232919, end_date=20250305T232920
[2025-03-05T23:29:20.649+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T23:29:20.661+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-05T23:29:20.662+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
